rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -570,7 +570,8 @@ String getJobHtml(
             @QueryParam("pending") @DefaultValue("true") boolean pending,
             @QueryParam("finished") @DefaultValue("true") boolean finished,
             @QueryParam("offset") @DefaultValue("0") int offset,
-            @QueryParam("limit") @DefaultValue("-1") int limit)
+            @QueryParam("limit") @DefaultValue("-1") int limit,
+            @QueryParam("sortparameters") SortSpecifierRestContainer sortParams)
```

Comment:
I followed the taxonomy of the other REST parameters so they're all named consistently, should I also rename them ?